### PR TITLE
feat: Implement insert content/notes attachments - EXO-73275

### PR DESCRIPTION
**Pull request by sofyenne**

### CODE REVIEW

File: apps/resources-wcm/src/main/webapp/eXoPlugins/attachFile/plugin.js [hunk 1]
```diff
@@ -6,20 +6,35 @@ CKEDITOR.plugins.add( 'attachFile', {
...
 
     // Create the toolbar button that executes the above command.
-    editor.ui.addButton( 'attachFile', {
+    const toolbar = {
       label: editor.lang.attachFile.buttonTooltip,
       command: 'attachFile',
       toolbar: 'insert'
-    });
+    };
+    editor.ui.addButton( 'attachFile', toolbar);
```
Comment (azayati) [inline apps/resources-wcm/src/main/webapp/eXoPlugins/attachFile/plugin.js]:
```suggestion
    editor.ui.addButton('attachFile', toolbar);
```

File: ecm-wcm-extension/src/main/webapp/WEB-INF/conf/wcm-extension/dynamic-container-configuration.xml [hunk 2]
```diff
@@ -117,6 +117,54 @@
         </object-param>
       </init-params>
     </component-plugin>
+    <!-- Add attachment app to the Content edition page -->
```
Comment (azayati) [inline ecm-wcm-extension/src/main/webapp/WEB-INF/conf/wcm-extension/dynamic-container-configuration.xml:120]: - Is there another way to inject the AttachmentApp without dynamic container with load-group
- you can add profiles="content"

**Review by azayati — CHANGES_REQUESTED** [review]

File: ecm-wcm-extension/src/main/webapp/WEB-INF/conf/wcm-extension/dynamic-container-configuration.xml (hunk 2 above)
Comment (sofyenne) [reply ecm-wcm-extension/src/main/webapp/WEB-INF/conf/wcm-extension/dynamic-container-configuration.xml:120]: Attachment app already injected for notes in this file!

**Review by azayati — APPROVED** [review]
LGTM

**Pull request merged**